tests/test_ldd.py

```diff
@@ -10,6 +11,9 @@
 
 import tests
 
+DIR = shutil.which("dir") or "/bin/dir"
```

Comment:
```suggestion
# Some distros like NixOS does not have binaries on /bin.
# Instead of hardcoding them, try to get them from PATH
# using shutil.which function.
DIR = shutil.which("dir") or "/bin/dir"
```